cloud-regionsrv-client.spec

```diff
@@ -16,7 +16,7 @@
 #
 
 
-%define base_version 9.2.1
+%define base_version 9.3.1
```

Comment:
There are going to be a number of changes to support AHBv3 and RMT as SCC relay, version bump is pre-mature.